docs/resources/database.md

```diff
@@ -1 +1,464 @@
-# :resources-database: Database
\ No newline at end of file
+# :resources-database: Database
+
+Database as a [Resource](/resources/) within DataOS is declared for creating a database within DataOS. It is treated as a first-class resource, encapsulating the fundamental unit for storing and organizing structured data using PostgreSQL. The classification of Databases as a resource type within the DataOS ecosystem allows for systematic management, provisioning, and utilization of databases in a standardized manner.
+
+<aside class=”callout”>
+
+ Database is a Workspace-level Resource type.
+
+</aside>
+
+
+<div class="grid cards" markdown>
+
+-   :material-card-bulleted-settings-outline:{ .lg .middle } **Key Features of Database**
+
+    ---
+
+    Understand the need of Database as a Resource in DataOS.
+
+    [:octicons-arrow-right-24: Key Features](#key-features-of-database)
+
+
+-   :material-list-box-outline:{ .lg .middle } **How to create a Database?**
+
+    ---
+
+    Learn how to create a database yaml specific to PostgreSQL.
+
+    [:octicons-arrow-right-24: Create a Database](#how-to-create-a-database?)
+
+-   :material-network-pos:{ .lg .middle } **Possible Errors**
+
+    ---
+
+    Discover potential errors and solutions for each
+
+    
+    [:octicons-arrow-right-24: Possible Errors](#possible-errors)
+
+-   :material-content-duplicate:{ .lg .middle } **Case Scenarios**
+
+    ---
+
+    Understand a end to end use case of using a database
+
+    [:octicons-arrow-right-24:  Possible errors](#case-scenarios)
+
+</div>
+
+
+
+## Key Features of Database
+
+### **Transactional Data Storage**
+
+A Database serves as a Resource, guaranteeing systematic storage of data generated and consumed by various applications within DataOS. The Beacon Service, through its exposed API endpoints, facilitates a seamless connection between applications and the Database, fostering a resourceful data storage hub.
+
+### **CRUD Operations for Data Assets**
+
+The Beacon Service enables CRUD operations (Create, Read, Update, Delete) on data assets stored in the PostgreSQL database. This functionality is pivotal for applications within DataOS to manipulate and interact with the underlying data efficiently.
+
+To learn more about the Beacon refer [Becon Resource](/resources/stacks/beacon/#configure-beacon-stack-specific-section)
+
```

Comment:
Remove this key features part and include these aspects in the introduction itself.